OpenSim/Examples/ControllerExample/ControllerExample.cpp

```diff
@@ -317,8 +318,11 @@ int main()
         manager.integrate( si );
 
         // Save the simulation results.
-        osimModel.printControlStorage( "tugOfWar_controls.sto" );
-        manager.getStateStorage().print( "tugOfWar_states.sto" );
+        auto controlsTable = osimModel.getControlsTable();
+        MOTFileAdapter::write(controlsTable, "tugOfWar_controls.mot");
```

Comment:
Currently the files are being written by MOTFileAdapter. I think it will incorrect to change the extension to `.sto`. My suggestion is to have another PR to introduce STOFileAdapter and change these file extensions there. Just creating a type alias STOFileAdapter will not make sense since we have to introduce that adapter as a separate class anyway.

If I can get the precise format of MOT files, I can start an another PR pretty quickly.